man/runc-list.8.md

```diff
@@ -4,12 +4,9 @@
 # SYNOPSIS
    runc list [command options] [arguments...]
 
-# DESCRIPTION
-   The default format is table.  The following will output the list of containers
-in json format:
-
-    # runc list -f json
-
 # OPTIONS
    --format, -f         select one of: table or json.
+                        The default format is table. The following will output
+                        the list of containers in json format:
+                        # runc list -f json
```

Comment:
I think we should keep help message simple and clear (one line per option would be good), elaborations can landed in man pages.